support/Pipelines/Bbh/Inspiral.py

```diff
@@ -16,6 +17,66 @@
 INSPIRAL_INPUT_FILE_TEMPLATE = Path(__file__).parent / "Inspiral.yaml"
 
 
+# These parameters come from empirically tested values in SpEC and SpECTRE
+def _control_system_params(
+    mass_left: float,
+    mass_right: float,
+    total_mass: float,
+    mass_ratio: float,
+    spin_magnitude_left: float,
+    spin_magnitude_right: float,
+) -> dict:
+    damping_time_base = 0.2
+    decrease_threshold_base = 2e-3
+    max_damping_timescale = 20.0
+    if spin_magnitude_left > 0.9 or spin_magnitude_right > 0.9:
+        damping_time_base = 0.1
+        decrease_threshold_base = 2e-4
+        max_damping_timescale = 10.0
+
+    params = {}
```

Comment:
(nitpick) For consistency with the code below I'd do this syntax, but that's just personal preference:

```py
return {
  "MaxDampingTimescale": max_damping_timescale,
  "KinematicTimescale": damping_time_base * total_mass,
  # ...
}
```

Have to precompute values used multiple times, but maybe that's better anyway.